components/NoteCard.tsx

```diff
@@ -0,0 +1,21 @@
+import { View, Text } from "react-native";
+
+export interface NoteCardProps {
+    noteTitle: string;
+    noteBody: string[];
+    bgColor: string;
+    textColor: string;
+}
+
+function NoteCard(props: NoteCardProps) {
+    return (
+        <View className={`rounded-xl px-4 py-5 min-w-[80%] bg-[${props.bgColor}] my-3`}>
+            <Text className="font-bold justify-start pb-2 text-[#F1E0C5] font-montserrat">{props.noteTitle}</Text>
```

Comment:
let's change the text color to also follow the custom textColor property (as seen in Figma)